.eslintignore

```diff
@@ -1,2 +1,4 @@
 **/*.js
 !plugins/Wallet/*
+!index.js
+!js/*
```

Comment:
there's probably an easier way to specify that...but I'm assuming we won't need an `.eslintignore` soon anyway